account_mass_reconcile_as_job/data/queue_job_channel_data.xml

```diff
@@ -0,0 +1,6 @@
+<odoo noupdate="1">
+    <record model="queue.job.channel" id="channel_mass_reconcile">
+        <field name="name">mass.reconcile.job</field>
```

Comment:
The previous channel used with the `job` decorator was named `mass_reconcile`, better to keep it for existing instance.